packages/e2e/lib/e2e.dart

```diff
@@ -184,4 +241,88 @@ class E2EWidgetsFlutterBinding extends LiveTestWidgetsFlutterBinding {
     );
     results[description] ??= 'success';
   }
+
+  /// Handle commands from WebDriver tests.
+  ///
+  /// WebDriver provides a set methods (ex: screenshot, resize, refresh) which
+  /// are useful for writing end-to-end tests on multiple browsers.
+  ///
+  /// e2e tests are sending [TestStatus] messages to state that they are ready
+  /// to run a WebDriver action ([TestStatus.wait_on_webdriver_command]) or
+  /// they are finished a web driver action
+  /// ([TestStatus.webdriver_command_complete])
+  ///
+  /// The tests running on the browser sends request these WebDriver operations
+  /// from e2e tests via replying with [WebDriverAction]s.
+  Future<Map<String, dynamic>> _callbackForWebDriver(
+      Map<String, String> params) async {
+    final String command = params['command'];
+    // Test status is added as an exta message.
+    final String extraMessage = params['message'] ?? '';
```

Comment:
Do we have this documented anywhere?

Would we be better off implementing some serialized type for this that could more clearly be used on both sides of the message?

Reply:
We can add a string to the requestData call for managing generic communication. Documentation is here: https://github.com/flutter/flutter/blob/master/packages/flutter_driver/lib/src/driver/driver.dart#L494

Are you recommending to use another command other than `RequestData`. I think that would make the handshake more complicated.